### PR TITLE
add entrypoint in custom image

### LINKED ISSUE #668: feat: Better support for custom base image
## Description

Support entrypoint and cache for custom base image

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingyang@tensorchord.ai>

* fix #668

### CODE REVIEW

**Comment by gaocegege** [comment]
Please at me when it is ready to review.

**Comment by kemingy** [comment]
> Please at me when it is ready to review.

PTAL

File: pkg/lang/ir/types.go [hunk 1]
```diff
@@ -53,6 +53,7 @@ type Graph struct {
 	Exec            []string
 	Copy            []CopyInfo
 	Mount           []MountInfo
+	entrypoint      []string
```
Comment (gaocegege) [inline pkg/lang/ir/types.go:56]: It's better to make it public, I am not sure if we use anywhere.
```suggestion
	Entrypoint      []string
```

Comment (kemingy) [reply pkg/lang/ir/types.go]: Fixed

**Review by gaocegege — APPROVED** [review]

**Pull request merged**